packages/web-twig/src/Resources/components/Breadcrumbs/README.md

```diff
@@ -0,0 +1,100 @@
+# Breadcrumbs
+
+This is Twig implementation of the [Breadcrumbs] component.
+
+Basic example usage:
+
+```html
+<Breadcrumbs items="items" />
```

Comment:
```suggestion
<Breadcrumbs items="{{ items }}" />
```

plus there is a missing definition of array of objects.

Reply:
There is no change that I suggest. Did you update the README? The current state does not seem right to me.

There should be an array of items defined and used via prop as I suggested.